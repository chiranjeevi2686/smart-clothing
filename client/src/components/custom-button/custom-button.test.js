import React from "react";
import { render } from '@testing-library/react';
import CustomButton from './custom-button.component';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

const defaultStore = {
};
const mockStore = configureStore();
describe('CustomButton component', () => {
    it('should render CustomButton', () => {
        const { asFragment } = render(<Provider store={mockStore(defaultStore)}>
            <CustomButton/>
        </Provider>)
        expect(asFragment()).toMatchSnapshot();
    })
})