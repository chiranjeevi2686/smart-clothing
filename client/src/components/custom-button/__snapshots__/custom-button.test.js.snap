// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton component should render CustomButton 1`] = `
<DocumentFragment>
  <button
    class="sc-bdfBwQ fgFZwo"
  />
</DocumentFragment>
`;
